sql/moz-fx-data-shared-prod/telemetry_derived/newtab_visits_v1/schema.yaml

```diff
@@ -221,6 +221,30 @@ fields:
   - name: pocket_recommendation_id
     type: STRING
     mode: NULLABLE
+  - mode: NULLABLE
+    name: sponsored_pocket_dismissals
+    type: INTEGER
+  - mode: NULLABLE
+    name: organic_pocket_dismissals
+    type: INTEGER
+  - mode: NULLABLE
+    name: pocket_thumbs_up
+    type: INTEGER
+  - mode: NULLABLE
+    name: pocket_thumbs_down
+    type: INTEGER
+  - mode: NULLABLE
+    name: pocket_received_rank
+    type: INTEGER
```

Comment:
`pocket_received_rank` would be a string since event extras are strings.  You would need to either cast this to an int in the query or drop and recreate the column in the deployed table